onnxruntime/core/framework/execution_frame.cc

```diff
@@ -292,8 +292,7 @@ ExecutionFrame::ExecutionFrame(const std::vector<int>& feed_mlvalue_idxs, const
             }
 
             // log size of activation. Keep it commented out for now to avoid log flooding.
-            // VLOGS(session_state_.Logger(), 1) << "Allocated memory for activations, size: "
-            //                                   << mem_patterns_->patterns[i].PeakSize();
+            printf("\n **** Allocated memory for activations, size: %zu ***\n", mem_patterns_->patterns[i].PeakSize());
```

Comment:
(nit):  replace with VLOGS or remove.

Reply:
Commented out, see above comment.

---
In reply to: [505110892](https://github.com/microsoft/onnxruntime/pull/4443#discussion_r505110892) [](ancestors = 505110892)